src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -72,7 +72,7 @@ export class AddBulkConcepts extends Component {
         params: { type, typeName, collectionName },
       },
     } = this.props;
-    const url = `${type}/${typeName}/collections/${collectionName}/references/`;
+    const url = `${type}/${typeName}/collections/${collectionName}/references/?cascade=sourcemappings`;
```

Comment:
What is the use of / before ?

Reply:
The API endpoints expect trailing slashes. It returns a `500` when there's no slash.